src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -1199,12 +1252,33 @@ void AlgorithmPaintToolBox::updateStroke(ClickAndMoveEventFilter * filter, medAb
 
     if(!view->contains(m_maskAnnotationData))
     {
+        m_maskAnnotationData->setMetaData("SeriesDescription", "mask");
         view->addLayer(m_maskAnnotationData);
+        foreach (medAbstractInteractor* interactor, view->layerInteractors(getWorkspace()->getSelectedLayerIndices()[0]))
+        {
+            if (interactor->identifier() == "medAnnotationInteractor")
+            {
+                foreach (medAbstractParameter* parameter, interactor->linkableParameters())
+                {
+                    if (parameter->name() == "Opacity")
+                    {
+                        qobject_cast<medDoubleParameter*>(parameter)->setValue(0.4);
+                    }
+                }
+            }
+        }
 
         // Update Mouse Interaction ToolBox
         view->setCurrentLayer(0);
         getWorkspace()->updateMouseInteractionToolBox();
     }
+
+    if ( slicingParameter )
+    {
+        slicingParameter->getSlider()->addTick(currentIdSlice);
```

Comment:
old ticks are not removed before?

Reply:
(a user can remove a ROI with only right-click "Erase", i don't know if it's seen by this tool)

Reply:
Done